### PR TITLE
Sinope floor thermostat quirks

### PR DESCRIPTION
**Pull request by claudegel**
Added support for Sinope TH11300ZB thermostats

### CODE REVIEW

**Comment by claudegel** [comment]
I think we can also add the TH1500ZB double pole and place it with the TH1123ZB and TH1124ZB. It work the same way

File: zhaquirks/sinope/thermostat.py [hunk 1]
```diff
@@ -164,3 +164,59 @@ class SinopeTH1400ZB(SinopeTechnologiesThermostat):
...
+                ],
+            }
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                INPUT_CLUSTERS: [
+                    Basic,
```
Comment (dmulcahey) [inline zhaquirks/sinope/thermostat.py:207]: @Adminiuga We have been using the id's when not replacing the clusters with different implementations. Curious on your thoughts for this? Should we use the id's or the clusters?

Comment (Adminiuga) [reply zhaquirks/sinope/thermostat.py:207]: Yep, should be using Cluster.ckuster_id here and use custom clusters for thermostat and manufacturer specific one

Comment (claudegel) [reply zhaquirks/sinope/thermostat.py:207]: So do we need to change something ?

File: zhaquirks/sinope/thermostat.py [hunk 2]
```diff
@@ -164,3 +164,59 @@ class SinopeTH1400ZB(SinopeTechnologiesThermostat):
...
+                    Scenes,
+                    UserInterface,
+                    TemperatureMeasurement,
+                    Metering,
+                    ElectricalMeasurement,
+                    Diagnostic,
+                    SinopeTechnologiesThermostatCluster,
+                    SinopeTechnologiesManufacturerCluster,
+                ],
+                OUTPUT_CLUSTERS: [Time, Ota, SINOPE_MANUFACTURER_CLUSTER_ID],
```
Comment (Adminiuga) [inline zhaquirks/sinope/thermostat.py:219]:
```suggestion
                    Basic.cluster_id,
                    Identify.cluster_id,
                    Groups.cluster_id,
                    Scenes.cluster_id,
                    UserInterface.cluster_id,
                    TemperatureMeasurement.cluster_id,
                    Metering.cluster_id,
                    ElectricalMeasurement.cluster_id,
                    Diagnostic.cluster_id,
                    SinopeTechnologiesThermostatCluster,
                    SinopeTechnologiesManufacturerCluster,
                ],
                OUTPUT_CLUSTERS: [
                    Time.cluster_id,
                    Ota.cluster_id,
                    SINOPE_MANUFACTURER_CLUSTER_ID,
                ],
```

Comment (Adminiuga) [reply zhaquirks/sinope/thermostat.py:219]: the clusters in the "replacement" that you don't actually replace should be identified by the `cluster_id`

Comment (claudegel) [reply zhaquirks/sinope/thermostat.py:219]: Should be the same for line 152 to 161 ?

Comment (claudegel) [reply zhaquirks/sinope/thermostat.py:219]: Should I do all those changes myself and push to PR ?

**Comment by claudegel** [comment]
What is LGTM ?

**Comment by Adminiuga** [comment]
Looks Good To Me

**Pull request merged**